Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM registry.redhat.io/ubi8/ubi AS builder
```

Comment:
Go for `FROM openshift/origin-release:golang-1.13 AS builder`

Reply:
> Using the CI imagestream means folks will need a pull secret to our CI registry.

Not really, it gets pulled from
https://hub.docker.com/r/openshift/origin-release/dockerfile/

Is there a similar builder UBI image? I'm fine with UBI, just want to avoid `yum install..`

Reply:
@sbose78 I think that's a good suggestion, thanks.